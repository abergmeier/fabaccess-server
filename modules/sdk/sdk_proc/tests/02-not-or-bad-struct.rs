

#[sdk_proc::module]
enum EnumModule {

}

#[sdk_proc::module]
struct UnitStructModule;

fn main() {}