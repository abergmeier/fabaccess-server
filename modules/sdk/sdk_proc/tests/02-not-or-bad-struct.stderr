error: Modules must be structs
 --> tests/02-not-or-bad-struct.rs:4:1
  |
4 | enum EnumModule {
  | ^^^^

error: Modules can't be unit structs
 --> tests/02-not-or-bad-struct.rs:9:24
  |
9 | struct UnitStructModule;
  |                        ^
